### DISCUSSION TITLE
针对可解析的pdf文件能否不用OCR，而根据bbox坐标直接从pdf文件中直接提取

### DISCUSSION THREAD

**Discussion by SopmmmodII** [PaddlePaddle/PaddleOCR#12578] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：
- 版本号/Version：Paddle：  PaddleOCR： 问题相关组件/Related components：版面还原/版面分析
- 运行指令/Command Code：
- 完整报错/Complete Error Message：

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no):yes

需要将非扫描版的pdf提取文章的组织结构还原成一棵树，如果使用OCR可能会增加耗时并且识别错误，能否利用版面分析结果中的bbox坐标直接通过读取原始pdf文件的方式获取内容？

**Comment by qingjiaozyn** [comment]
同样的问题

**Comment by gggdroa** [comment]
蹲一个解答

**Comment by SWHL** [comment]
这个是可以的。参考博客:[pdfplumber和pdfminer.six提取PDF中文本行内容及对应坐标](https://blog.csdn.net/shiwanghualuo/article/details/124886446?ops_request_misc=%257B%2522request%255Fid%2522%253A%2522171714620616800197065944%2522%252C%2522scm%2522%253A%252220140713.130102334.pc%255Fblog.%2522%257D&request_id=171714620616800197065944&biz_id=0&utm_medium=distribute.pc_search_result.none-task-blog-2~blog~first_rank_ecpm_v1~rank_v31_ecpm-2-124886446-null-null.nonecase&utm_term=pdf&spm=1018.2226.3001.4450)
